deblur/deblurring.py

```diff
@@ -36,7 +36,10 @@ def get_sequences(input_seqs):
     seqs = [Sequence(label, seq) for label, seq in input_seqs]
 
     if len(seqs) == 0:
-        raise ValueError("No sequences found!")
+        logger = logging.getLogger(__name__)
+        logger.warn('No sequences found in fasta file!')
```

Comment:
in a parallel context, shouldn't this logging message and the ones below denote the sample?